spec/views/idv/inherited_proofing/agreement.html.erb_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+describe 'idv/inherited_proofing/agreement.html.erb' do
+  let(:flow_session) { {} }
+  let(:sp_name) { nil }
+  let(:locale) { nil }
+
+  before do
+    allow(view).to receive(:decorated_session).and_return(@decorated_session)
+    allow(view).to receive(:flow_session).and_return(flow_session)
+    allow(view).to receive(:url_for).and_return('https://www.example.com/')
+  end
+
+  it 'renders the Continue button' do
+    render template: 'idv/inherited_proofing/agreement'
+
+    expect(rendered).to have_button(t('inherited_proofing.buttons.continue'))
+  end
+
+  describe 'I18n' do
+    before do
```

Comment:
I thought we agreed to remove most text rendering tests... especially I18n tests.